src/nbl/asset/interchange/IAssetLoader.cpp

```diff
@@ -35,6 +35,24 @@ void IAssetLoader::IAssetLoaderOverride::insertAssetIntoCache(SAssetBundle& asse
         m_manager->insertAssetIntoCache(asset, ASSET_MUTABILITY_ON_CACHE_INSERT);
 }
 
+bool IAssetLoader::IAssetLoaderOverride::handleRestore(core::smart_refctd_ptr<IAsset>&& _chosenAsset, SAssetBundle& _bundle, SAssetBundle& _reloadedBundle, uint32_t _restoreLevels)
+{
+    if (_bundle.getContents().size() != _reloadedBundle.getContents().size())
+        return false;
+
+    auto dummies = _bundle.getMutableContents();
```

Comment:
why the call to `getMutableContents` its not like you'll be changing the pointers values stores in the bundle (the contents of the dynamic array)